diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatGradlePlugin.kt

```diff
@@ -0,0 +1,52 @@
+package org.cqfn.diktat.plugin.gradle
+
+import com.pinterest.ktlint.reporter.plain.PlainReporter
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.artifacts.ExternalModuleDependency
+
+/**
+ * Plugin that configures diktat and registers tasks to run diktat
+ */
+@Suppress("unused", "MagicNumber")
+class DiktatGradlePlugin : Plugin<Project> {
+    /**
+     * @param project a gradle [Project] that the plugin is applied to
+     */
+    override fun apply(project: Project) {
+        val diktatExtension = project.extensions.create(DIKTAT_EXTENSION, DiktatExtension::class.java)
+        diktatExtension.inputs = project.fileTree("src").apply {
```

Comment:
src? what if it will be test/src/blabla

Reply:
This is the default value, it can be overridden from `diktat` extension.